### PR TITLE
Added 2018-08-28-reactiveui-succession-update.md

### PR DESCRIPTION
**Pull request by RLittlesII**

### CODE REVIEW

File: input/blog/2018-08-28-reactiveui-succession-update.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+We are trying to make the contributor experience better.  [RFC 18](https://github.com/reactiveui/rfcs/issues/18) is about enforcing a consistent coding style across the entire code base.  We are going to configure analyzers that will enforce the [Core Fx](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md#c-coding-style) rules.  This will remove the friction of contributors having to check to see if their changes are compliant with our rules before submitting a Pull Request!
```
Comment (glennawatson) [inline input/blog/2018-08-28-reactiveui-succession-update.md]: Should you mention the comments being an advantage to the end user btw?

Comment (RLittlesII) [reply input/blog/2018-08-28-reactiveui-succession-update.md]: Added

**Comment by glennawatson** [comment]
When the CI has finished feel free to check in at your leisure

**Comment by RLittlesII** [comment]
@glennawatson Requires another review as I added verbiage on the xml comments.

**Pull request merged**